ui/scss/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "studio-scss",
+  "version": "1.0.0",
+  "description": "SASS for studio",
+  "private": true,
+  "main": "index.js",
```

Comment:
Just a question, how do I tell yarn to run this file?

Reply:
You may create a scripts section on the `package.json` like so:
```
  "scripts": {
    "build": "node index.js"
  },
```
Then you'd do `yarn build` to build it through yarn.

You may also directly call `node index.js` which is what I did to run/test during my testing.

Reply:
I have to check to see if that is possible with the maven plugin, because it uses a custom node binary